docs/proposals/pepxxx_wheel_variant_support.md

```diff
@@ -772,6 +777,9 @@ variant properties.
 The order of variant properties is defined first by the order of namespaces, then the order of features in the namespace
 and then the order of property values in a feature.
 
+Q: It's not clear where these properties are defined.  Is it in the `pyproject.toml` for the consuming package
+or somewhere else?
```

Comment:
I don't think "ordering" is the right section to describe this, but yeah, may need clarifying somewhere that they're defined at build time in build backend-defined manner and put into `variant.json`.

Reply:
Included in https://github.com/wheelnext/wheelnext/pull/94